docs/source/conf.py

```diff
@@ -18,10 +18,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'GraphQL Compiler'
-# The copyright string gets displayed as "2017-present Kensho Technologies, LLC." in the browser,
-# (with one dot at the end). If we add a dot after LLC in the copyright string, it will appears as
-# "2017-present Kensho Technologies, LLC.."
-copyright = '2017-present Kensho Technologies, LLC'
+copyright = '2017-present Kensho Technologies, LLC.'
```

Comment:
With the new theme this appears as '2017-present Kensho Technologies, LLC.', (without the double dots).